lib/stdlib/doc/src/erl_tar.xml

```diff
@@ -104,21 +105,18 @@
     <title>Limitations</title>
     <list type="bulleted">
       <item>
-        <p>For maximum compatibility, it is safe to archive files with names
-          up to 100 characters in length. Such tar files can generally be
-          extracted by any <c>tar</c> program.</p>
-      </item>
-      <item>
-        <p>For filenames exceeding 100 characters in length, the resulting tar
-          file can only be correctly extracted by a POSIX-compatible <c>tar</c>
+        <p>For filenames exceeding 100 bytes in length (not counting use of the prefix field),
+          the resulting tar file can only be correctly extracted by a USTAR-compatible <c>tar</c>
```

Comment:
Certainly not in common use, but there may be very old systems out there still running a version of `tar` which only supports the v7 standard - anything newer can use the prefix field to split the path. The key limitation is that the actual name of a file (not including it's path) is still limited to 100 characters in any version of `tar` not supporting PAX. The modern versions of GNU/BSD tar support PAX though, so that limitation only applies to older versions.